packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -181,53 +259,66 @@ class MemoryBodyState extends State<MemoryBody>
     controller.memorySourcePrefix = mediaWidth > verboseDropDownMinimumWidth
         ? MemoryScreen.memorySourceMenuItemPrefix
         : '';
-    return Column(
-      children: [
-        Row(
-          mainAxisAlignment: MainAxisAlignment.spaceBetween,
-          children: [
-            _buildPrimaryStateControls(textTheme),
-            const Expanded(child: SizedBox(width: denseSpacing)),
-            _buildMemoryControls(textTheme),
-          ],
-        ),
-        SizedBox(
-          height: 50,
-          child: MemoryEventsPane(eventChartController),
-        ),
-        SizedBox(
-          child: MemoryVMChart(vmChartController),
-        ),
-        controller.isAndroidChartVisible
-            ? SizedBox(
-                height: defaultChartHeight,
-                child: MemoryAndroidChart(androidChartController),
-              )
-            : const SizedBox(),
-        const SizedBox(height: defaultSpacing),
-        Row(
-          children: [
-            TabBar(
-              labelColor: textTheme.bodyText1.color,
-              isScrollable: true,
-              controller: tabController,
-              tabs: MemoryBody.memoryTabs,
-            ),
-            const Expanded(child: SizedBox()),
-          ],
-        ),
-        const SizedBox(width: defaultSpacing),
-        Expanded(
-          child: TabBarView(
-            physics: defaultTabBarViewPhysics,
-            controller: tabController,
+    return RawKeyboardListener(
+      focusNode: FocusNode(),
+      onKey: (RawKeyEvent event) {
+        if (event.isKeyPressed(LogicalKeyboardKey.escape)) {
```

Comment:
Can we integrate into the general Flutter focus system instead of listening for escape ourselves?

Reply:
Maybe but after playing with Focus, FocusNode, and FocusScope doesn't seem as straight forward because I have no TextField.  Maybe I could create a hidden text field but that seems ugly too. I'll ad a TODO.